tmt-app/src/Components/BobaPath.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import {Container, Typography} from '@mui/material';
+import ForwardArrow from '../Images/Forward.png';
+
+
+function BobaPath(){
+    return(
+        <Container>
+            <Container 
+                sx={{
+                    width:"100%", 
+                    display:"flex", 
+                    alignItems:"center",
+                    textAlign:"center"
+                }}
+            >
+                <Typography variant="CustomSubHeading" sx={{color:"BobaHighlight.main"}}>Student Projects</Typography>
+                <img 
+                    style={{width:"24px", margin: "0 16px"}}
+                    src={ForwardArrow} 
+                    alt="ForwardArrow"/>
+                <Typography variant="CustomSubHeading" sx={{color:"BobaHighlight.main"}}>Major Picker</Typography>
+                </Container>
+        </Container>
+    )
+}
+
+export default BobaPath;
```

Comment:
These are typically called "breadcrumbs"
https://mui.com/material-ui/react-breadcrumbs/

Material has a breadcrumb component to provide the address and hierarchy of the page with some some out of the box styling as well. The separator is also included by default and can be overridden with icons https://mui.com/material-ui/react-breadcrumbs/#custom-separator. we should consider something like the `ForwardArrow` be converted to svg over a PNG or we check in with design to see if they intended to use MaterialIcons

Reply:
Didn't know the specific term for this, but I will change to mui breadcrumb component since it would more efficient.